### ISSUE TITLE
Alt text available in item/fileset thumbnails

### ISSUE THREAD

**Issue by laritakr** [samvera/hyrax#7378]
## Summary

Handled by https://github.com/samvera/hyrax/pull/7375

Alt text can be added to a fileset by:
- editing the fileset to add alt text to its metadata
- importing a fileset in a Bulkrax CSV. The file would need to be in a fileset row with the alt text included. (The alt text cannot be added to a file added via a work row)
